README.md

```diff
@@ -24,7 +24,7 @@ Version 3.0 of OctopusDSC supports Octopus Deploy 4.x and above with backwards c
 ## Installation
 
 ### [PowerShellGet](https://technet.microsoft.com/en-us/library/dn807169.aspx) / PowerShell 5 (recommended)
-1. Install PowerShellGet from [PowerShell Gallery](https://www.powershellgallery.com/GettingStarted)
```

Comment:
Following the guide, I noticed some of these links no longer work. (the vagrant.io tried to open from within the github domain).

We could consider a little more detail concerning TeamCity env variables.